source/agora/consensus/protocol/Nominator.d

```diff
@@ -235,6 +246,12 @@ extern(D):
         }
     }
 
+    /// Trigger network cathup of unknown txs
+    private void txCatchupTask ()
+    {
+        this.network.getUnknownTXs(this.ledger);
+    }
+
```

Comment:
What I mentioned last time was trying to use this timer:
https://github.com/bosagora/agora/blob/31b9fd794b703e98b8188f57b13ddce242b51b17/source/agora/node/FullNode.d#L460-L461

Otherwise we risk having two timers doing the same thing that trigger at the same time.
You might need to split what the timer do, but we really shouldn't have two timers doing this.

Reply:
I can create a new timer for `getUnknownTX` in `FullNode` and pass the timers to the`Nominator` class.

Reply:
Pushed an update with new timer for `tx catchup` and pass it to `Nominator`. I expect I will need to split some of the changes into a separate commit. But want to see if it passes Ci first.